lib/action_cable/channel/base.rb

```diff
@@ -168,7 +204,6 @@ def action_signature(action, data)
           end
         end
 
-
```

Comment:
Ah, sorry. Thought it was between a method scope change (like going from public to private), which is split by two newlines in Action Cable.